runtime/lua/vim/_defaults.lua

```diff
@@ -143,6 +143,29 @@ do
     end
     vim.keymap.set({ 'o' }, 'gc', textobject_rhs, { desc = 'Comment textobject' })
   end
+
+  --- Defaults maps for LSP functions.
+  ---
+  --- These are mapped unconditionally to avoid confusion. If no server is attached, or if a server
+  --- does not support a capability, an error message is displayed rather than exhibiting different
+  --- behavior.
+  do
+    vim.keymap.set('n', 'crn', function()
+      vim.lsp.buf.rename()
+    end, { desc = 'vim.lsp.buf.rename()' })
+
+    vim.keymap.set('n', 'cra', function()
+      vim.lsp.buf.code_action()
+    end, { desc = 'vim.lsp.buf.code_action()' })
+
+    vim.keymap.set('n', 'gr', function()
+      vim.lsp.buf.references()
+    end, { desc = 'vim.lsp.buf.references()' })
+
+    vim.keymap.set('i', '<C-S>', function()
```

Comment:
`i_CTRL-G_CTRL-P` (mnemonic: "parameters") is available, opens the door for future `i_CTRL-G` mappings.

Reply:
Not crazy about the double mapping, but we can revisit. We have a little time before this makes it to a release. If we did do the `<C-G>` prefix, I have a preference for `<C-G><C-S>` (mnemonic: "signature") since that can be typed with one hand (at least on QWERTY).

`<C-H>` might also be worth considering, though some people might use the default "backspace" behavior.